scripts/build-data.js

```diff
@@ -69,6 +69,7 @@ const envMap = {
   iojs: "node3.3",
   node64: "node6",
   node65: "node6.5",
+  node76: "node7.6",
```

Comment:
I believe https://github.com/babel/babel-preset-env/pull/86 is supposed to fix this but didn't really look at it @graingert